doc/help/commands.asciidoc

```diff
@@ -261,7 +261,7 @@ Start hinting.
  - With `spawn`: The executable and arguments to spawn.
  `{hint-url}` will get replaced by the selected
  URL.
- - With `userscript`: The userscript to execute.
+ - With `userscript`: The userscript to execute. Either store the userscript in your data directory or use an absolute path. 
```

Comment:
This file is autogenerated by `scripts/dev/src2asciidoc.py` - so please don't edit it directly, edit the docstring for the functions in `qutebrowser/browser/hints.py` and `qutebrowser/browser/commands.py`.